nf_core/pipeline-template/conf/base.config

```diff
@@ -54,4 +54,7 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
+    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
+        cache = false
```

Comment:
Why is this necessary? (I know it was like that before, but i never understood why)

Reply:
So MultiQC is always re-run regardless of whether the pipeline is cached or not.

Reply:
I think there were issues around it not being re-run properly when it had to be but not 💯